### PR TITLE
(fix) Issue #33

### PR DESCRIPTION
**Pull request by Al-Pragliola**
Incorrect reporting of service status in failed state

### CODE REVIEW

File: web-client/src/Stores/SystemMDStatus.ts [hunk 1]
```diff
@@ -52,7 +52,12 @@ export class SystemMDStatus {
 
 	public toggleServiceAndWatchStatus = async () => {
 		const startOrStop = this.active_status === 'active' ? 'stop' : 'start';
+		runInAction(() => {
+				this.active_status = this.active_status === 'active' ? 'deactivating' : 'activating';
+			},
+		);
 		await this.startOrStopService(startOrStop);
+		await sleep(1000);
```
Comment (angelbarrera92) [inline web-client/src/Stores/SystemMDStatus.ts:60]: I'm not a frontend expert (far from that...)
Why this sleep?

**Comment by Al-Pragliola** [comment]
Issue: after clicking on the button start, two things happen:
1) Call on start service
2) Get the status of this service

what was happening is the status call was faster than the service failing so it was showing incorrectly the status "Active", what I did to fix is insert a loading state between 1) and 2) with a fixed delay of 1 second

File: web-client/src/Stores/SystemMDStatus.ts (hunk 1 above)
Comment (Al-Pragliola) [reply web-client/src/Stores/SystemMDStatus.ts:60]: commented below

 |
V

Comment (Al-Pragliola) [reply web-client/src/Stores/SystemMDStatus.ts:60]: Issue: after clicking on the button start, two things happen:

Call on start service
Get the status of this service
what was happening is the status call was faster than the service failing so it was showing incorrectly the status "Active", what I did to fix is insert a loading state between 1) and 2) with a fixed delay of 1 second

**Pull request merged**